### PR TITLE
Update email compile related docs

### PR DESCRIPTION
**Pull request by tomaszszymanski129**
As we discussed we should provide pre-compiled email templates instead of forcing user to compile them on start. Should be merged along with backend changes on https://github.com/mirumee/saleor/pull/5260.

### CODE REVIEW

File: docs/customization/emails.md [hunk 1]
```diff
@@ -21,7 +21,18 @@ Emails defined in `*.email` files include their HTML versions by referencing the
 
 Source emails use [MJML](https://mjml.io/) and must be compiled to HTML before use.
 
-To compile emails, run:
+### Before you start
+
+To compile the emails you need to have node.js installed.
```
Comment (maarcingebala) [inline docs/customization/emails.md]:
```suggestion
To compile the emails you need to have Node.js installed.
```

File: docs/customization/emails.md [hunk 2]
```diff
@@ -21,7 +21,18 @@ Emails defined in `*.email` files include their HTML versions by referencing the
...
+Version 10 or later is required. See the [installation instructions](https://nodejs.org/en/download/package-manager/).
+
+You also need to install frontend dependencies first, run:
```
Comment (maarcingebala) [inline docs/customization/emails.md]: I suggest a little rephrase to make it more natural

```suggestion
You also need to install frontend dependencies first. To do that run:
```

File: docs/customization/emails.md [hunk 3]
```diff
@@ -23,10 +23,10 @@ Source emails use [MJML](https://mjml.io/) and must be compiled to HTML before u
 
 ### Before you start
 
-To compile the emails you need to have node.js installed.
+To compile the emails you need to have Node.js installed.
 Version 10 or later is required. See the [installation instructions](https://nodejs.org/en/download/package-manager/).
```
Comment (NyanKiyoshi) [inline docs/customization/emails.md]: Node JS 8 or greater actually (please check, but AFAIK that's the MJML requirement) which would be easier for non-rolling release distribution users.

Comment (maarcingebala) [reply docs/customization/emails.md]: Node.js 8 is no longer supported: https://github.com/nodejs/Release#release-schedule. I guess it's fine as it is.

**Pull request merged**